### ISSUE TITLE
Angular Ivy update

### ISSUE THREAD

**Issue by kisimediaDE** [dstelljes/luxon-angular#28]
Hey :-) 
when I serve the app I get the following message in the terminal: 

```
Processing legacy "View Engine" libraries:
luxon-angular [es2015/esm2015] (github:dstelljes/luxon-angular)
Encourage the library authors to publish an Ivy distribution.
```

Greetings,
Simon

**Comment by ruizmarc** [comment]
Library should be updated with Angular 13 and in tsconfig for production should add:

```
{
  "angularCompilerOptions": {
    "compilationMode": "partial"
  }
}
```

**Comment by dstelljes** [comment]
Thank you both; this is released with 5.0.0.

**Comment by kisimediaDE** [comment]
Thanks for the fast response and solution 👍 
I appreciate this very much!!!!
